templates/cli/lib/type-generation/languages/dart.js.twig

```diff
@@ -0,0 +1,68 @@
+/** @typedef {import('../attribute').Attribute} Attribute */
+const { AttributeType } = require('../attribute');
+const { LanguageMeta } = require("./language");
+
+class Dart extends LanguageMeta {
+  getType(attribute) {
+    let type = "";
+    switch (attribute.type) {
+      case AttributeType.STRING:
+      case AttributeType.EMAIL:
+      case AttributeType.DATETIME:
+      case AttributeType.ENUM:
+        type = "String";
```

Comment:
Could we generate actual enums for enum attributes (for languages than have them)?